app/models/graph.rb

```diff
@@ -1,5 +1,13 @@
+require 'timeout'
+
 module Graph
   class << self
+    def query(query_string, variables: {}, context: {}, timeout: nil)
+      Timeout.timeout(timeout) do
```

Comment:
😯  http://www.mikeperham.com/2015/05/08/timeout-rubys-most-dangerous-api/

Does that apply here? (That's the reason I didn't use the built-in API for `graphql-ruby`'s timeout middleware)